zerver/tests/test_muting_users.py

```diff
@@ -182,3 +182,16 @@ def test_messages_from_muted_user_marked_as_read(self) -> None:
         pm_to_user = self.send_personal_message(muted_user, user, "Spam in PM")
         user_private_um = list(UserMessage.objects.filter(user_profile=user, message=pm_to_user))[0]
         self.assertTrue(user_private_um.flags.read)
+
+        pm_to_iago = self.send_personal_message(muted_user, iago, "Spam in PM")
+
+        url = "/api/v1/users/me/muted_users/{}".format(muted_user.id)
```

Comment:
Add a comment before this line explaining what API call we're about to do / what we're about to test.